### DISCUSSION TITLE
增加mod管理菜单里的排序功能

### DISCUSSION THREAD

**Discussion by mclvlc** [Meloong-Git/PCL#1495] [讨论]
比如按启用状态排序，按装载时间排序，按大小排序...

**Comment by Chiloven945** [comment]
这个得看API能不能做到了……
如果新功能可以交issue

**Reply by mclvlc** [reply]
哟西谢谢我是想弄到新功能里的

**Comment by WForst-Breeze** [comment]
### 注意：这里是**讨论**板块，不是**提问**板块，close 了先，下次注意哦

启用状态：#679
其他两个的话，您要是想要可以去交个新功能提案。不过我个人认为直接打开 Mod 文件夹不是更方便嘛……
![image](https://user-images.githubusercontent.com/110760354/227699221-a2fb89ea-6b71-48ad-8ecb-f60284f9eab0.png)

**Comment by Chiloven945** [comment]
才发现我看成了mod搜索界面（
抱歉……
